CHANGES.md

```diff
@@ -1,3 +1,26 @@
+## edge-19.7.5
+
+* CLI
+  * Improved the installation report produced by the `linkerd check` command
+    to include the control plane pods' live status
+  * Added the `--all-namespaces` (`-A`) option to the `linkerd get`,
```

Comment:
The change was just adding the `-A` shortcode, this sounds more like we added `--all-namespaces` support in general, maybe something like:
```
* Added `-A` shortcode for the `--all-namespaces` option in `linkerd get`, `linkerd edges`, and `linkerd stat`.
```